app/utils/tap.js

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-const doublePressDelay = 300;
-
-export function preventDoubleTap(func) {
+export function preventDoubleTap(func, delayTime) {
```

Comment:
One small suggestion here. We could simplify this slightly by specifying the default value when defining the parameter.
```js
export function preventDoubleTap(func, doublePressDelay = 300) {
```